gammapy/modeling/parameter.py

```diff
@@ -843,12 +894,16 @@ def __init__(
         min=np.nan,
         max=np.nan,
         error=0,
+        scale_method="scale10",
```

Comment:
This is really sad that this important class has no docstring... A reference to some tutorials could be also appreciated by users.
Even if it is not the scope of this PR, one could add that now OR make quickly an issue to update the documentation.